app/views/admin/edition_tags/edit.html.erb

```diff
@@ -33,6 +33,9 @@
         </div>
       </div>
 
+      <div class="content content-bordered" data-module="breadcrumb-preview">
```

Comment:
The intention is for this widget to be javascript only, but the form as a whole should work without javascript.

If you don't have javascript, you will see the same browse paths after saving on the document overview page. This widget displays the same information but you get feedback before you click save.